### ISSUE TITLE
Dragging a pinned tab switches focus to the pinned tab

### ISSUE THREAD

**Issue by Smile4ever** [eoger/tabcenter-redux#365]
I'm in an unpinned tab. I drag the first pinned tab to the second position. The focus is changed from my unpinned tab to the pinned tab. I just wanted to reorder pinned tabs, I didn't want to switch to it.

* Mouseup should be used to switch to a tab, not mousedown

**Comment by Keith94** [comment]
Don't the native tabs use the same mousedown behavior? As noted by @eoger: https://github.com/eoger/tabcenter-redux/pull/277#issuecomment-362180737

**Comment by Smile4ever** [comment]
@Keith94 Yes, indeed, I just tested and the native tab bar does the same. Is there a workaround or an option to get the behaviour I want?

**Comment by Keith94** [comment]
It would have to be implemented, I think. https://github.com/eoger/tabcenter-redux/pull/277/commits/ea0384a9af8c7da4efbe1024b549de3351344256 was rejected because it doesn't follow Firefox behavior.
